packages/libs/react-ui/src/components/Grid/Grid.css.ts

```diff
@@ -28,3 +28,123 @@ export const gapVariants = styleVariants({
   // eslint-disable-next-line @typescript-eslint/naming-convention
   '3xl': [sprinkles({ gridGap: '$3xl' })],
 });
+
+export const rowSpanVariants = styleVariants({
+  // eslint-disable-next-line @typescript-eslint/naming-convention
+  1: [{ gridRow: 'span 1' }],
+  // eslint-disable-next-line @typescript-eslint/naming-convention
+  2: [{ gridRow: 'span 2' }],
+  // eslint-disable-next-line @typescript-eslint/naming-convention
+  3: [{ gridRow: 'span 3' }],
+  // eslint-disable-next-line @typescript-eslint/naming-convention
+  4: [{ gridRow: 'span 4' }],
+  // eslint-disable-next-line @typescript-eslint/naming-convention
+  5: [{ gridRow: 'span 5' }],
+  // eslint-disable-next-line @typescript-eslint/naming-convention
+  6: [{ gridRow: 'span 6' }],
+  // eslint-disable-next-line @typescript-eslint/naming-convention
+  7: [{ gridRow: 'span 7' }],
+  // eslint-disable-next-line @typescript-eslint/naming-convention
+  8: [{ gridRow: 'span 8' }],
+  // eslint-disable-next-line @typescript-eslint/naming-convention
+  9: [{ gridRow: 'span 9' }],
+  // eslint-disable-next-line @typescript-eslint/naming-convention
+  10: [{ gridRow: 'span 10' }],
+  // eslint-disable-next-line @typescript-eslint/naming-convention
+  11: [{ gridRow: 'span 11' }],
+  // eslint-disable-next-line @typescript-eslint/naming-convention
+  12: [{ gridRow: 'span 12' }],
+});
+
+const columnCount: Record<number, number> = {
+  // eslint-disable-next-line @typescript-eslint/naming-convention
+  1: 1,
+  // eslint-disable-next-line @typescript-eslint/naming-convention
+  2: 2,
+  // eslint-disable-next-line @typescript-eslint/naming-convention
+  3: 3,
+  // eslint-disable-next-line @typescript-eslint/naming-convention
+  4: 4,
+  // eslint-disable-next-line @typescript-eslint/naming-convention
+  5: 5,
+  // eslint-disable-next-line @typescript-eslint/naming-convention
+  6: 6,
+  // eslint-disable-next-line @typescript-eslint/naming-convention
+  7: 7,
+  // eslint-disable-next-line @typescript-eslint/naming-convention
+  8: 8,
+  // eslint-disable-next-line @typescript-eslint/naming-convention
+  9: 9,
+  // eslint-disable-next-line @typescript-eslint/naming-convention
+  10: 10,
+  // eslint-disable-next-line @typescript-eslint/naming-convention
+  11: 11,
+  // eslint-disable-next-line @typescript-eslint/naming-convention
```

Comment:
in the library we use `$1` as a convention we use in the lib.
Can we use that here?

Reply:
My reasoning for using plain numbers instead of $-prefixed ones is that, in VE variables, the numbers represent a count for the measuring ($3 = 0.75rem = 12px) whereas in here the numbers represent the weight, out of 12, that each column should take.